src/i18n/locales/en/translation.json

```
@@ -351,7 +351,12 @@
       "text_miner_fee_in_satspervbyte_exact": "{{ value }} sats/vByte",
       "text_miner_fee_in_satspervbyte_randomized": "{{ min }} - {{ max }} sats/vByte",
       "text_miner_fee_in_targeted_blocks_one": "High priority estimate (next block)",
-      "text_miner_fee_in_targeted_blocks_other": "Using estimate for inclusion in next {{ count }} blocks"
+      "text_miner_fee_in_targeted_blocks_other": "Using estimate for inclusion in next {{ count }} blocks",
+      "label_eligible_utxos": "Eligible UTXOs",
+      "description_eligible_utxos": "One or more of the following UTXOs will be included in the transaction: ",
+      "label_selected_utxos": "Selected UTXOs",
+      "description_selected_utxos_one": "The following UTXO will be included the transaction: ",
+      "description_selected_utxos_other": "The following UTXOs will be included the transaction: "
```

```suggestion
      "description_selected_utxos_multiple": "The following UTXOs will be included the transaction: "
```
nit, maybe?
same for subtile_other

---

This is just how i18next works by convention: https://www.i18next.com/translation-function/plurals

---

Ohh my bad